analyses/focal-cn-file-preparation/01-add-ploidy-cnvkit.Rmd

```diff
@@ -70,6 +95,8 @@ add_xy_df <- add_ploidy_df %>%
     chrom == "chrX" & germline_sex_estimate == "Female" & copy.num == tumor_ploidy ~ "neutral"
   ))
 
+
+
```

Comment:
```suggestion
```

Reply:
Not implemented, change is now outdated.